fl4health/clients/fed_prox_client.py

```diff
@@ -98,7 +98,8 @@ def compute_loss(
         self, preds: Dict[str, torch.Tensor], features: Dict[str, torch.Tensor], target: torch.Tensor
     ) -> Losses:
         """
-        Computes loss given predictions of the model and ground truth data.
+        Computes loss given predictions of the model and ground truth data. Adds to objective by including
+        proximal loss which is the L2 norm between the initial and final weights of local training.
```

Comment:
I'm going to be a pedantic mathematician and have you lower-case l2 here. L^2/L_2 norms operate on functions not vectors lol

Reply:
hahahah don't ever not be a pedantic mathematician, its good to know these notation conventions